### PR TITLE
Coordinates

### PR DESCRIPTION
**Pull request by gllisch**
Added this because its where I am here #446

### CODE REVIEW

File: gllisch.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: gllisch
```
Comment (gllisch) [inline gllisch.json]: Is this the right name?

**Comment by gllisch** [comment]
👍 Perfect

**Comment by gllisch** [comment]
@githubteacher closes #446

File: gllisch.json (hunk 1 above)
Comment (jr-cologne) [inline gllisch.json]: Yes, that should be right.

**Comment by gllisch** [comment]
Thanks @jr-cologne for the help

**Comment by jr-cologne** [comment]
You are welcome. 👍

**Pull request merged**